python/pyspark/ml/tests.py

```diff
@@ -655,6 +656,20 @@ def test_nested_pipeline_persistence(self):
             except OSError:
                 pass
 
+    def test_write_property(self):
+        lr = LinearRegression(maxIter=1)
+        path = tempfile.mkdtemp()
+        lr_path = path + "/lr"
+        lr.write.save(lr_path)
```

Comment:
I added similar test before. However, that test doesn't capture the change in the inherit class (i.e., missing adding property in inherit class) and Yanbo suggested adding a concrete test for testing the functionality. Let me know your final decision. Thanks!

Miao